test-case/check-sof-logger.sh

```diff
@@ -41,57 +45,103 @@ ldcFile=$(find_ldc_file) || die ".ldc file not found!"
 
 dlogi "Found file: $(md5sum "$ldcFile"|awk '{print $2, $1;}')"
 
+# These filenames are kept for backward-compatibility
+# DMA trace
 data_file=$LOG_ROOT/logger.data.log
+# stderr
 error_file=$LOG_ROOT/logger.error.log
```

Comment:
The web cannot render .log file, so it automatically download it for you, that's what you see in the internal web, please rename it to *.txt.